include/ht_light_component.h

```diff
@@ -80,9 +81,22 @@ namespace Hatchit {
 
         private:
 
+            /**
+            * \brief Sets the light mesh with the appropriate type for current renderer.
+            * \param meshFile File name of mesh to load
+            * \param meshFile File name of material to load
+            */
```

Comment:
Can you move any documentation into the implementation file as described in our code spec?